pyproject.toml

```diff
@@ -47,6 +47,9 @@ includes = "src"
 [tool.pytest.ini_options]
 asyncio_mode = "auto"
 testpaths = ["tests"]
+markers = [
```

Comment:
cleaner to see our custom markers properly registered amongst other ones: :slightly_smiling_face: 
![Screenshot from 2022-05-11 10-12-01](https://user-images.githubusercontent.com/722388/167814296-cd40f435-b9d1-4491-9825-a6e1f0df2ca4.png)